src/functions/Mock.ps1

```diff
@@ -1529,7 +1529,7 @@ function Get-DynamicParametersForCmdlet {
             $Parameters = @{ }
         }
 
-        $cmdlet = & $SafeCommands['New-Object'] $command.ImplementingType.FullName
+        $cmdlet = ($command.ImplementingType)::new()
```

Comment:
More readable with explicit New-Object?

Reply:
maybe, but nothing a comment could not solve.